Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM golang:latest AS builder
```

Comment:
Minor: I'd call this file e.g. `Dockerfile.validate` or something to be more clear.